physical/azure/azure_test.go

```diff
@@ -16,11 +16,14 @@ import (
 	"github.com/hashicorp/vault/sdk/physical"
 )
 
-func environmentForCleanupClient(name string) (azure.Environment, error) {
-	if name == "" {
-		return azure.EnvironmentFromName("AzurePublicCloud")
+func environmentForCleanupClient(name string, armUrl string) (azure.Environment, error) {
+	if name != "" {
+		return azure.EnvironmentFromName(name)
 	}
-	return azure.EnvironmentFromName(name)
+	if armUrl != "" {
```

Comment:
Should this check come first (for consistency with `azure.go`)?

Reply:
Good point, I've adjusted the check to be similar to the one in `azure.go`.